ortools_vendor/patches/fix_clang_issue.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/ortools/constraint_solver/routing_filter_committables.h b/ortools/constraint_solver/routing_filter_committables.h
+index 5249064..1d38f41 100644
+--- a/ortools/constraint_solver/routing_filter_committables.h
++++ b/ortools/constraint_solver/routing_filter_committables.h
+@@ -58,7 +58,7 @@ template <typename T>
+ class CommittableArray {
+  public:
+   // Makes a vector with initial elements all committed to value.
+-  CommittableArray<T>(size_t num_elements, const T& value)
++  CommittableArray(size_t num_elements, const T& value)
```

Comment:
@codebot this is the required change, AFAIK this is just not valid c++ code, but clang seems to allow it.